crates/cdk-cli/src/sub_commands/mint.rs

```diff
@@ -22,6 +22,9 @@ pub struct MintSubCommand {
     /// Currency unit e.g. sat
     #[arg(default_value = "sat")]
     unit: String,
+    /// Quote description
+    #[arg(default_value = None)]
```

Comment:
T
```suggestion
 #[serde(skip_serializing_if = "Option::is_none")]
```

Default None is implicit we can replace it with skip if none.